docs/backup_and_restore.rst

```diff
@@ -75,57 +75,42 @@ the SecureDrop servers.
 
 .. code:: sh
 
-   source .venv/bin/activate
-   cd install_files/ansible-base
-   ansible -m ping all
+   ssh app uptime
 
 If this command fails (usually with an error like "SSH Error: data could not be
 sent to the remote host. Make sure this host can be reached over ssh"), you need
 to debug your connectivity before proceeding further. Make sure:
 
-* Ansible is installed (``which ansible`` should return a path instead of "not found").
-
-  * Ansible should be automatically installed by the Tails auto-configuration
-    for SecureDrop. If it is not, you probably need to re-run
-    ``./securedrop-admin tailsconfig``. See
-    :doc:`configure_admin_workstation_post_install` for detailed instructions).
+* Ansible is installed. If it is not, see
```

Comment:
This isn't really a troubleshooting step for `ssh app uptime` right? I guess it's fine to leave in since they _do need_ Ansible to be installed anyway